example/example_opt.jl

```diff
@@ -120,19 +120,26 @@ end
 
 # Set lower and upper limits
 # DESIGN VARIABLES
-#             AR    Alt(ft)  Cl     Λ     λs  λt   hboxo   hboxs   rcls    rclt     Tt4CR   iepihc iepif
-lower      = [7.0 , 20000.0, 0.40, 10.0, 0.1, 0.1, 0.10,   0.10,   0.1,    0.1,     700.0,  6,      0]
-upper      = [12.0, 60000.0, 0.65, 40.0, 1.0, 1.0, 0.15,   0.15,   1.4,    1.0,     2000.0, 15,     10] 
+
+lower      = [9.0 , 0.53, 25.0, 10000.0, 0.65, 0.1,  0.125,    0.125,    0.9,   0.7, 1400.0, 10.0,  1.25, 2.98]
+upper      = [11.0, 0.60, 30.0, 10900.0, 0.85, 0.4,  0.15,   0.15,   1.3,   1.0, 1650.0, 15.0, 2.0, 3.02] 
+#initial    = [10.987, 0.598, 27.58, 10015.46, 0.85, 0.1587, 0.1449, 0.126, 1.0295, 0.993, 1649.359, 13.346, 1.8436, 3.0 ]
+initial    = [10.5, 0.57, 26.0, 10668.0, 0.7, 0.25, 0.1268, 0.1266, 1.238, 0.9, 1580.0, 12.0, 1.685, 3.0 ]
+
+
+# #             AR    Alt(ft)  Cl     Λ     λs  λt   hboxo   hboxs   rcls    rclt     Tt4CR   iepihc iepif
```

Comment:
Why are you leaving the old params commented out? May be better to just delete them